### PR TITLE
Feature/integration tests/adds new user

### PR DESCRIPTION
**Pull request by genti94**
finished add-new-user.js

extended gulpfile

extended cms.js

extended globals.js

### CODE REVIEW

File: cms/tests/frontend/integration/helpers/cms.js [hunk 1]
```diff
@@ -3,10 +3,18 @@ var globals = require('../settings/globals');
 
 module.exports = function (casperjs) {
     return {
-        login: function () {
+        /**
+         * Logs in with the given parameters
+         *
+         * @public
+         * @param {Object} [credentials=globals.credentials]
+         * @param {String} opts.username
```
Comment (vxsx) [inline cms/tests/frontend/integration/helpers/cms.js]: should be `credentials.username`

File: cms/tests/frontend/integration/settings/globals.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ module.exports = {
     adminUrl: 'http://localhost:8000/en/admin/login/',
     adminLogoutUrl: 'http://localhost:8000/en/admin/logout/',
     adminPagesUrl: 'http://localhost:8000/en/admin/cms/page/',
+    adminPageUsers: 'http://localhost:8000/en/admin/auth/user/',
```
Comment (vxsx) [inline cms/tests/frontend/integration/settings/globals.js]: should probably be consistent and name it `adminUsersUrl`

File: cms/tests/frontend/integration/settings/globals.js [hunk 3]
```diff
@@ -27,6 +28,10 @@ module.exports = {
     },
     user: {
         // TODO add more data for the new user profile
-        firstName: 'test-first-name'
+        firstName: 'test-first-name',
```
Comment (vxsx) [inline cms/tests/frontend/integration/settings/globals.js]: should move it out of `globals` and put it in `addNewUser` instead

**Comment by vxsx** [comment]
addNewUser.js is actually missing :)

File: cms/tests/frontend/integration/addNewUser.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+casper.test.setUp(function (done) {
+    casper.start()
+        .then(cms.login())
+        .then(cms.addPage({ title: 'home' }))
+        .run(done);
+});
+
+casper.test.tearDown(function () {
+});
```
Comment (vxsx) [inline cms/tests/frontend/integration/addNewUser.js]: should remove the page and logout in the teardown

File: cms/tests/frontend/integration/addNewUser.js [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+                // delete button gets clicked
+                .waitForSelector('#user_form', function () {
+                    test.assertExists('#user_form', 'User Form has been loaded');
+                    this.click('.deletelink');
+                })
+
+                // confirming that the user gets deleted
+                .waitForSelector('.delete-confirmation', function () {
+                    test.assertExists('.delete-confirmation', 'Delete button clicked confirmed');
+                    this.click('input[type="submit"]')
```
Comment (vxsx) [inline cms/tests/frontend/integration/addNewUser.js]: missing semicolon

Comment (vxsx) [inline cms/tests/frontend/integration/addNewUser.js]: will figure out why travis doesn't fail on lint error later

**Comment by FinalAngel** [comment]
@genti94 what is the status here? can we land this soon?

**Pull request merged**